assets/js/settings/blocks/constants.js

```diff
@@ -32,6 +32,11 @@ export const DISPLAY_PRICES_INCLUDING_TAXES = getSetting(
 );
 export const PRODUCT_COUNT = getSetting( 'productCount', 0 );
 export const ATTRIBUTES = getSetting( 'attributes', [] );
+export const ENABLE_SHIPPING_CALCULATION = getSetting(
+	'isShippingCalculatorEnabled',
+	true
+);
+export const HIDE_SHIPPING_COST = getSetting( 'isShippingCostHidden', false );
```

Comment:
I think we should name these constants `IS_SHIPPING_CALCULATOR_ENABLED` and `IS_SHIPPING_COST_HIDDEN`, that would keep them consistent with the settings and make it clear they are booleans.

Reply:
I agree, let's use the same convention for naming these, so it's clear that they are a value (not an action) and boolean type.